app/javascript/packages/address-search/index-spec.tsx

```diff
@@ -5,7 +5,7 @@ import { setupServer } from 'msw/node';
 import { rest } from 'msw';
 import type { SetupServer } from 'msw/node';
 import { SWRConfig } from 'swr';
-import AddressSearch, { ADDRESS_SEARCH_URL, LOCATIONS_URL } from './address-search';
+import AddressSearch, { ADDRESS_SEARCH_URL, LOCATIONS_URL } from './index';
```

Comment:
I think `index` is implicit with the path.

```suggestion
import AddressSearch, { ADDRESS_SEARCH_URL, LOCATIONS_URL } from './';
```